tutorials/messengers/telegram/4_conditions.py

```diff
@@ -32,15 +31,10 @@
 In our Telegram module, we adopted the system of filters
 available in the `pytelegrambotapi` library.
```

Comment:
The tutorials definitely need a complete rewrite

I think there should be max 5 tutorials. Could you make a proposal about the tutorials that should be included in the new version?